Cannot find module 'broccoli-merge-trees'

After 1246cdc46161ca01f54b476c58f8e95ac36b13a6,
ember serve fails with:
Cannot find module 'broccoli-merge-trees'.
(using ember-cli 0.2.3)

Workaround:
```
npm install --save-dev broccoli-merge-trees
npm install --save-dev broccoli-funnel
```

--- a/package.json
+++ b/package.json
@@ -26,8 +26,6 @@
   "author": "Sam Selikoff",
   "devDependencies": {
     "broccoli-asset-rev": "^2.0.0",
-    "broccoli-funnel": "^0.2.3",
-    "broccoli-merge-trees": "^0.2.1",
     "chai": "2.0.0",
     "ember-cli": "0.2.0",
     "ember-cli-app-version": "0.3.2",
@@ -54,5 +52,8 @@
   "ember-addon": {
     "configPath": "tests/dummy/config"
   },
-  "dependencies": {}
+  "dependencies": {
+    "broccoli-funnel": "^0.2.3",
+    "broccoli-merge-trees": "^0.2.1"
+  }
 }

